tests/lib/hardware/openstack_libcloud.py

```diff
@@ -201,6 +201,12 @@ def get_ssh_ip(self):
         # NOTE(jhesketh): For now, just use the last floating IP
         return self._floating_ips[-1].ip_address
 
+    def add_data_disk(self, capacity):
+        # TO BE IMPLEMENTED
+        logger.warn('add_data_disk() not implemented for'
+                    'openstack backend yet')
+        pass
```

Comment:
This can be removed given that there is now the warning

Reply:
Done